BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.java

```diff
@@ -151,7 +151,10 @@ public void run() {
                         request.id = Integer.valueOf(ids[i]);
                         request.event = events[i];
                         request.timestamp = Long.valueOf(timestamps[i]);
-                        request.metadata = new JSONObject(cursor.getString(cursor.getColumnIndex(META_JSON)));
+                        int columnIndex = cursor.getColumnIndex(META_JSON);
+                        if (columnIndex >= 0) {
```

Comment:
The build was failing locally for me on this index potentially being 0... I'm not sure what changed to cause this error, but adding a check here resolved it.

Reply:
can I get some more details on this bit? I'm a little confused about it. What is this doing exactly?

Reply:
I think this lint build error was related to the beta Android Studio version I have... We will probably need to make this change in the future but I'm going to revert it for now since it's not related to this PR.

Reply:
:+1: